lib/src/jvmMain/kotlin/cash/z/ecc/android/random/SecureRandom.kt

```diff
@@ -0,0 +1,8 @@
+package cash.z.ecc.android.random
+
+actual class SecureRandom {
+    val secureRandom = java.security.SecureRandom()
```

Comment:
Due to #13, I suspect that we might eventually need a `SecureRandomFactory` and a `suspend fun new(): SecureRandom` method so that we can move the SecureRandom construction off the main thread.

No need to change this now, but noting that such a change is probably coming in case it alters how you think about this.